repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/TestProjectUtils.scala

```diff
@@ -5,14 +5,15 @@ import java.util.UUID
 import cromwell.pipeline.datastorage.dto.{ Project, ProjectId, UserId }
 
 object TestProjectUtils {
-  val userUuid: String = UUID.randomUUID().toString
-  val projectUuid: String = UUID.randomUUID().toString
 
+  private def randomUuidStr: String = UUID.randomUUID().toString
+  def getDummyProjectId: ProjectId = ProjectId(randomUuidStr)
+  private def getDummyUserId: UserId = UserId(randomUuidStr) // todo move this method to the TestUserUtils
```

Comment:
Since you did add `getDummyUserId` to a `TestUserUtils` class, there is no need in this method.
`TestUserUtils.getDummyUserId` on line 14 and that's all.

Reply:
fixed